keyboards/adpenrose/mine/keymaps/via/rules.mk

```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```

Comment:
May make sense to put this at the keyboard level, rather than on multiple keymaps.

Reply:
Even though not all the keymaps use the layout that defines all the possible keys for the layouts supported?

Reply:
drashna is referring to the `LTO_ENABLE = yes`. It can be added to `info.json` as
```json
"build": {
    "lto": true
}
```

Reply:
Oh, that makes more sense haha. I'll do it ASAP.